### ISSUE TITLE
[Question]: Graph PaddleNLP Information Extraction Model Evaluation 

### ISSUE THREAD

**Issue by jonyllamado** [PaddlePaddle/PaddleNLP#8816]
### 请提出你的问题

Hello. Is there a way to visualize the model evaluation of PaddleNLP Information Extraction (Documents)? I would like to use this to interpret the results of my machine learning model whether it is underfitting or overfitting. I would like to be able to plot the loss and accuracy graphs per steps

**Comment by jonyllamado** [comment]
Update: I found out how to graph it. For your reference:

**Reading information from your log file**
```
from visualdl import LogReader

reader = LogReader(file_path='./path/to/your/.log_file)
```
```
tags = reader.get_tags()
print(tags)
```
```
my_loss = reader.get_data('scalar', 'train/loss')
print(my_loss)
```


**Writing your log file to the accepted format of VisualDL**
```
from visualdl import LogWriter

with LogWriter(logdir="./my_log") as writer:
   for step in range(len(my_loss)):
        writer.add_scalar(tag="train/loss", step=step, value=my_loss[step].value, walltime=my_loss[step].timestamp)
```

**Visualize Graph**
`! visualdl --logdir ./my_log --port 8080`
